chalicelib/checks/lifecycle_checks.py

```diff
@@ -94,39 +94,42 @@ def patch_file_lifecycle_status(connection, **kwargs):
             upload_key, bucket=raw_bucket, print_error=False
         ):
             file_bucket = raw_bucket
-        if not file_bucket:
-            action_logs["error"].append(f"Cannot patch file {uuid}: not found on S3")
-            continue
 
         try:
             s3_tag = lifecycle_utils.lifecycle_status_to_s3_tag(new_lifecycle_status)
-            if s3_tag:
+
+            if not s3_tag:
+                raise Exception(f"Could not determine S3 tag for file {uuid}")
+
+            if file_bucket:
                 my_s3_util.set_object_tags(
                     key=upload_key,
                     bucket=file_bucket,
                     tags=s3_tag,
                     merge_existing_tags=True,
                 )
-
-                if not is_extra_file:
-                    today = datetime.date.today().strftime("%Y-%m-%d")
-                    patch_dict = {
-                        "s3_lifecycle_status": new_lifecycle_status,
-                        "s3_lifecycle_last_checked": today,
-                    }
-
-                    file_status = lifecycle_utils.lifecycle_status_to_file_status(
-                        new_lifecycle_status
-                    )
-                    if (
-                        file_status == lifecycle_utils.ARCHIVED
-                        or file_status == lifecycle_utils.DELETED
-                    ):
-                        patch_dict["status"] = file_status
-
-                    ff_utils.patch_metadata(patch_dict, uuid, key=my_auth)
             else:
-                raise Exception(f"Could not determine S3 tag for file {uuid}")
+                action_logs["logs"].append(f"Cannot tag file {uuid}: not found on S3")
+                # In this case, set everything to 'deleted' in the metadata
+                new_lifecycle_status = lifecycle_utils.DELETED
```

Comment:
My concern with this is that files to be uploaded by users would also be set to a deleted status if they haven't uploaded a file within 2 weeks, and such gaps are known to occur, especially when there are different people to accession the case and to upload the files.

Could we perhaps exclude the `uploading` status from the file search queries as well (see other comment)? An alternative would be to revert this change and just exclude the `to be uploaded by workflow` status from the file search queries as well if you want to be a little more conservative here.